Getting started sections (#2169)
giantswarm/docs: src/content/getting started/prepare-your-provider-account/aws/_index.md

---

> title: Prepare your provider account for AWS
> linkTitle: Prepare your AWS account
> description: The actions need to be done to prepare your AWS account for starting to build your cloud-native developer platform.

Reviewer: ```suggestion
title: Prepare your AWS account
description: Prepare your AWS account to start building your cloud-native developer platform with Giant Swarm.
```

[repeated for every provider]

I had a better sentence yesterday but lost the comment draft – feel free to improve any of the suggestions.
